akka-persistence-typed/src/test/scala/docs/akka/persistence/typed/BasicPersistentBehaviorCompileOnly.scala

```diff
@@ -200,9 +200,7 @@ object BasicPersistentBehaviorCompileOnly {
           emptyState = State(),
           commandHandler = CommandHandler.command { cmd =>
             context.log.info("Got command {}", cmd)
-            Effect.persist(cmd).thenRun { state =>
-              context.log.info("event persisted, new state {}", state)
-            }
+            Effect.none
```

Comment:
Probably has the same in the corresponding Java sample that needs an update as well?

Reply:
I am glad that you pointed this out but interestingly no commandHandler() method exists on the Java side. 

You may have a look at it here: https://doc.akka.io/docs/akka/2.6/typed/persistence.html?#accessing-the-actorcontext

Reply:
Ah, thanks for checking!

Reply:
You're most welcome :)